test/1.2/misc/devinfo.dml

```diff
@@ -15,8 +15,8 @@ bank b[j in 0..3][k in 0..1] "bank desc" {
     }
     register r1[2] size 4 @ 18 + $i * 4 {
         parameter documentation = "reg1 doc";
-        field f0[7:3] "f0 desc";
-        field f1[2:1];
+        field f0[9:5] "f0 desc";
+        field f1[4:1];
```

Comment:
why not an array here, like in 1.4? 1.2 is forward-compatible with the syntax `field f1[i in 0..1] @ [2*i+2:2*i+1]`

Reply:
I dunno, I didn't write the original code! I'll do it if no issue crops up.

Reply:
ah, I misread the diff in 1.4 code (I thought you moved from non-array to array there). Never mind then.